examples/helpers/tiled.rs

```diff
@@ -259,9 +312,28 @@ pub fn process_loaded_maps(
                         };
 
                         let map_type = match tiled_map.map.orientation {
-                            tiled::Orientation::Hexagonal => {
-                                TilemapType::Hexagon(HexCoordSystem::Row)
-                            }
+                            tiled::Orientation::Hexagonal => match tiled_map.map.stagger_axis {
+                                StaggerAxis::X => match tiled_map.map.stagger_index {
+                                    StaggerIndex::Even => {
+                                        // TODO: Why Even is Odd?
```

Comment:
I believe this is because of how bevy handles Y coordinates. In tiled map editor Y down goes into the positive and in bevy Y down goes into the negative. When we load the tiles in we end up flipping the entire tilemap along the Y axis using: `mapped_y = tiled_map.map.height - 1 - y;`. Because of this we also need to "flip" the stagger.

Reply:
That makes sense. I'm not sure what to do about that other than add a comment to explain the situation.